models/product-type/src/product-type/product-type-draft/presets/sample-data-goodstore/bedding-bundle.ts

```diff
@@ -0,0 +1,57 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import {
+  AttributeDefinitionDraft,
+  AttributeLocalizableTextType,
+} from '../../../../index';
+import type { TProductTypeDraftBuilder } from '../../../types';
+import * as ProductTypeDraft from '../../index';
+
+const beddingBundle = (): TProductTypeDraftBuilder =>
+  ProductTypeDraft.presets
+    .empty()
+    .key('bedding-bundle')
+    .name('Bedding Bundle')
+    .description('Product type used to bundle bedding components')
+    .attributes([
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('product-description')
+        .type(AttributeLocalizableTextType.random())
+        .label(LocalizedString.presets.empty()['en-GB']('product-description'))
```

Comment:
We'll need to include `en-US` for all localized string occurrences

Reply:
Do we need `en-DE` for all as well?

Reply:
See b2202a33d7e2e7dd06bcb925d78998737df9499a for included US translations